### ISSUE TITLE
Please  modify  either the cmakelist.txt or the readme.md  of the YOLOv8 ONNX Runtime C++ Example

### ISSUE THREAD

**Issue by craii** [ultralytics/ultralytics#23125]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

The onnxruntime is required to be pplaced in the YOLOV8-ONNXRUNTIE-CPP folder according to the cmakelist and the version of it must be 1.15.1

<img width="1512" height="949" alt="Image" src="https://github.com/user-attachments/assets/4a752c3a-46df-42d3-a567-3f2f426b6af8" />

BUT  implied in the readme,  the path to the onnxruntime could be set to anywhere we want. 
Please  modify  either the cmakelist.txt or the readme.md  of the YOLOv8 ONNX Runtime C++ Example. IT GOT MY 5 MINUTES WASTED ..
<img width="2545" height="1309" alt="Image" src="https://github.com/user-attachments/assets/6604642c-7459-4608-82bf-fdb348471e7d" />

### Environment

Ultralytics 8.3.246 🚀 Python-3.11.14 torch-2.9.0 CPU (Apple M3)
Setup complete ✅ (8 CPUs, 24.0 GB RAM, 424.3/460.4 GB disk)

OS                     macOS-26.2-arm64-arm-64bit
Environment            Darwin
Python                 3.11.14
Install                pip
Path                   /Users/eliascheung/Documents/yolo11/yoloenv/lib/python3.11/site-packages/ultralytics
RAM                    24.00 GB
Disk                   424.3/460.4 GB
CPU                    Apple M3
CPU count              8
GPU                    None
GPU count              None
CUDA                   None

numpy                  ✅ 2.2.6>=1.23.0
matplotlib             ✅ 3.10.7>=3.3.0
opencv-python          ✅ 4.12.0.88>=4.6.0
pillow                 ✅ 12.0.0>=7.1.2
pyyaml                 ✅ 6.0.3>=5.3.1
requests               ✅ 2.32.5>=2.23.0
scipy                  ✅ 1.16.3>=1.4.1
torch                  ✅ 2.9.0>=1.8.0
torch                  ✅ 2.9.0!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision            ✅ 0.24.0>=0.9.0
psutil                 ✅ 7.1.3>=5.8.0
polars                 ✅ 1.35.2>=0.20.0
ultralytics-thop       ✅ 2.0.18>=2.0.18

### Minimal Reproducible Example

read the issue

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @craii, thank you for your interest in Ultralytics 🚀! This is an **automated response** to help route your report quickly, and an **Ultralytics engineer will also assist soon** 🙌

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it. In this case, since the report references the YOLOv8 ONNX Runtime C++ example, an MRE would ideally include:
-Exact example folder/file path you’re using (and which branch/commit)
-Your `CMakeLists.txt` changes (if any) and the exact `cmake`/build commands you ran
-The expected behavior vs. actual behavior (copy/paste the full terminal output)
-Where your `onnxruntime` directory is located on disk (relative and absolute paths)

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by glenn-jocher** [comment]
Thanks for flagging this — the README and `CMakeLists.txt` for the YOLOv8 ONNX Runtime C++ example are currently inconsistent (the CMake assumes a fixed, in-folder `onnxruntime` layout and was validated against ONNX Runtime `1.15.1`), and we’ll update one side so the “custom path” guidance matches reality; in the meantime you can either symlink/copy your ORT folder into the expected project subfolder or edit the ORT path in `CMakeLists.txt`, and if you paste the exact `CMakeLists.txt` lines here as text (not screenshots) we can patch it precisely (export side reference: [ONNX export docs](https://docs.ultralytics.com/integrations/onnx/)).
